loader/src/sources/albertsons/index.js

```diff
@@ -206,18 +213,69 @@ async function getData(states) {
       return formatted;
     })
     .filter((location) => states.includes(location.state));
+
+  // Adult & Pediatric vaccines at the same locations get different listings in
+  // the Albertsons API, so we need to group and merge them.
+  const groups = _.groupBy(formatted, (location) => {
+    return (
+      location.external_ids.find(
+        (id) => id[0] === "albertsons_store_number"
+      )?.[1] ?? location.name
+    );
+  });
+
+  return Object.values(groups).map((group) => {
+    if (group.length === 1) return group[0];
+
+    const adult = group.find((l) => l.meta.booking_url_adult);
+    const pediatric = group.find((l) => l.meta.booking_url_pediatric);
+
+    if (!adult || !pediatric) {
+      warn(
+        "Trying to merge locations other than an adult and pediatric!",
+        group
+      );
+    }
+
+    const result = Object.assign({}, ...group);
+    result.meta = {
+      ...adult.meta,
+      booking_url_adult: adult.booking_url,
+      booking_url_pediatric: pediatric.booking_url,
```

Comment:
In the weird case where `!adult || !pediatric` one of these two lines will blow up. I think maybe you should just exit the function rather than `warn`ing above.

Reply:
Oh, geez, I know I had a `return null` there at one point. Will fix!